General/IcebergOnWindows.md

```diff
@@ -0,0 +1,43 @@
+### Long file names in Windows
+
+Its error message goes by, “Failed to stat file ‘filename’: The filename or extension is too long”.
+
+The Windows problem with long filenames is a constraint of [libgit](https://libgit2.org/libgit2/#HEAD) library that is used in Pharo (Iceberg accesses git through this exact library). It occurs because the FileTree format tends to have such long paths and was one of the main drivers in creating the Tonel format. This error can occur, for example, when you are trying to clone your Pharo fork on Github into Pharo local. There are two possible ways to fix it:
+
+1. Make path to Pharo Image short.
+2. Convert repo to Tonel format — in Iceberg there is a menu item under one of the Extras menus.
+
+Here’s more about them:
+
+#### 1. Make path to the Pharo Image shorter
+
+If you are simply using a standalone Pharo [download](http://pharo.org/download/#standalone), you can just move your image and VM a few folders up. However, if you are using [Pharo Launcher](http://pharo.org/download/), you have to access the settings, to change where it stores images it will create in the future, as well as move all the existing images and VMs up. Here’s how to do it.
+
+In Pharo Launcher, in the bottom right corner you can see a button ‘open settings’ — click on it.
```

Comment:
Maybe a screenshot to illustrate that?

Reply:
1. not sure how to add a screenshot
2. not sure how to edit this file anymore, i was only able to merge the suggestions you made